packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Application} from 'probot';
+import {GitHubAPI} from 'probot/lib/github';
+import * as yaml from 'js-yaml';
+import * as util from 'util';
+
+const CONFIGURATION_FILE_PATH = '.bots/blunderbuss.yml';
```

Comment:
Other bots I've seen use the `.github/name_of_bot` pattern for capturing this.  Thoughts on using the common `.github` dir?